scripts/cm_test_all_sandia

```diff
@@ -587,62 +587,36 @@ elif [ "$MACHINE" = "inouye" ]; then
 
   SPACK_HOST_ARCH="+a64fx"
 elif [ "$MACHINE" = "weaver" ]; then
-  MODULE_ENVIRONMENT="source /etc/profile.d/modules.sh"
+  # Use the legacy env for now until all modules are part of the new system
+  MODULE_ENVIRONMENT="source /projects/ppc64le-pwr9-rhel8/legacy-env.sh"
```

Comment:
@e10harvey this is the change to make the legacy env available, the rest of the changes are cleanup to remove unsupported compilers